baseplate/events/queue.py

```diff
@@ -80,6 +80,25 @@ def set_field(self, key, value, obfuscate=False):
             obfuscated_payload = self.payload.setdefault("obfuscated_data", {})
             obfuscated_payload[key] = value
 
+    def unset_field(self, key):
+        """Remove the key from payload if it exists.
+        :param key:
```

Comment:
I think what's going on with the spellchecker is that rST is pretty picky about line spacing (and even if it weren't it'd be good to be consistent with other docstrings in this package). The param should also actually be documented. How about the following for docs for consistency sake:

```python
"""Remove a field from the event if it exists.

:param str key: The name of the field.
:return: The value of the removed field or :py:data:`None`.

"""
```

this function should also probably support the `obfuscated` flag like `get_field` does.